services/src/main/scala/cromwell/services/womtool/models/MetaValueElementJsonSupport.scala

```diff
@@ -0,0 +1,40 @@
+package cromwell.services.womtool.models
+
+import io.circe.{Encoder, Json}
+
+import wom.callable.MetaValueElement
+import wom.callable.MetaValueElement._
+
+object MetaValueElementJsonSupport {
+
+  // We only implement the encoder, because currently, limited query functionality is required
+  // for the meta sections. Even in the encoder, we only support the type, and not the actual value.
+  implicit val metaValueElementEncoder: Encoder[MetaValueElement] = new Encoder[MetaValueElement] {
```

Comment:
I'm surprised you need this... when would we be converting meta objects into json?